knowledge_repo/converters/ipynb.py

```diff
@@ -46,13 +58,17 @@ def from_file(self, filename):
         c = Config()
         # c.ExtractOutputPreprocessor.extract_output_types = set()
         c.ExtractOutputPreprocessor.output_filename_template = 'images/{unique_key}_{cell_index}_{index}{extension}'
+        c.NbConvertBase.display_data_priority = ['application/javascript', 'text/html', 'text/markdown',
+                                                 'image/svg+xml', 'text/latex', 'image/png', 'image/jpeg',
+                                                 'text/plain']
 
         nb = nbformat.read(filename, as_version=4)
 
         dl = DictLoader({'full.tpl': TEMPLATE})
         md_exporter = MarkdownExporter(config=c, extra_loaders=[
                                        dl], template_file='full.tpl')
         (body, resources) = md_exporter.from_notebook_node(nb)
+        body = body.replace('</div>\n<script', '</div>\n\n<script')
```

Comment:
Why is this necessary? Once it is rendered in HTML, why do we need multiple newlines?